react/src/util/opReturn.ts

```diff
@@ -82,24 +95,29 @@ function getDataPushdata(data: string, disablePaymentId=false) {
 // `68656c6c6f20776f726c64` - data payload, 'hello world' as ASCII encoded to hex
 // `08` - pushdata for the optional paymentId (paymentId), signifying this tx has 8 bytes of paymentId data
 // `0102030405060708` - The 8-byte paymentId (paymentId)
+//
+// Result: 0450415900001668656c6c6f20776f726c64080102030405060708
 
-export function parseOpReturnProps(
+export function encodeOpReturnProps(
```

Comment:
Should link to the spec in the documentation for this function (i.e. just a comment above would be good)
`//https://github.com/Bitcoin-ABC/bitcoin-abc/blob/master/doc/standards/paybutton.md`

Reply:
I've redone the tests in https://github.com/PayButton/paybutton/pull/353; since I had to change the implementation. I added this comment there.